src/components/Reviews/index.jsx

```diff
@@ -30,7 +32,7 @@ class Reviews extends React.Component {
     this.setState({ index: nextIndex });
   };
 
-  componentDidMount() {}
+  componentDidUpdate() {}
```

Comment:
you don't need either of these functions to move to the next index. Your event handlers are updating the state and that should be enough. The problem is that in those event handlers, whatever you are updating your state with, has a problem. Take a closer look.